marker: {
  size: 0.15, 
  id_list: [100]
}
